_includes/tutorials/connect-sink-timestamp/ksql/markup/dev/create-sink-connector.adoc

```diff
@@ -0,0 +1,7 @@
+Now create a JDBC sink connector with the code below. Notice no transforms will be necessary to the timestamp field because it is already a supported `TIMESTAMP` type.
```

Comment:
"it is" -- what is it?

Reply:
addressed in the latest commit https://github.com/confluentinc/kafka-tutorials/pull/760/commits/fe37cdbb1eb4538e6474ccb6e0f8f38e283efa77